src/CodeViewer/Viewer.cpp

```diff
@@ -239,9 +239,13 @@ void Viewer::DrawSampleCounters(QPaintEvent* event) {
                                fontMetrics().height()};
       painter.setPen(kLineNumberForegroundColor);
 
+      double function_percentage =
+          (code_report_->GetNumSamplesInFunction()
+               ? static_cast<double>(num_samples_in_line) / code_report_->GetNumSamplesInFunction()
+               : 0.);
```

Comment:
Could you make a small static function out of this?

Reply:
I modified DoubleToPercentageString for FractionToPercentageString. Seems to be easier to read.

Reply:
Even better.